schema.go

```diff
@@ -128,7 +128,7 @@ func GenerateSchema(options AvroSchemaGenOptions) (MessageSchema, error) {
 		ceType = fmt.Sprintf("%sNotification", baseName)
 	}
 	ceType = strcase.ToCamel(ceType) // to be sure if it's provided by the user
-	namespace := strcase.ToDelimited(options.Namespace, '.')
+	namespace := options.Namespace
```

Comment:
We should use the `getNamespace` function here it was missing. And add a check on the regexp of the namespace specification